ui/src/helpers/form-helpers.js

```diff
@@ -873,28 +873,38 @@ let formHelpers = function() {
             e.preventDefault();
 
             // Open the upload dialog with our options.
-            self.namespace.openUploadForm(
+            openUploadForm(
                 {
-                    oldMediaId: mediaId,
-                    widgetId: widgetId,
-                    updateInAllChecked: uploadFormUpdateAllDefault,
-                    trans: playlistAddFilesTrans,
-                    upload: {
-                        maxSize: $(this).data().maxSize,
-                        maxSizeMessage: $(this).data().maxSizeMessage,
-                        validExtensionsMessage: translations.validExtensions + ': ' + dialog.find('form').data().validExtensions,
-                        validExt: validExtensions
-                    }
-                }, {
-                    main: {
-                        label: translations.done,
-                        className: 'btn-primary',
-                        callback: function() {
-                            if(typeof self.namespace.timeline.resetZoom === 'function') {
-                                self.namespace.timeline.resetZoom();
+                    url: libraryAddUrl,
+                    title: uploadTrans.uploadMessage,
+                    animateDialog: false,
+                    initialisedBy: "library-upload",
```

Comment:
I was not sure about this...

Reply:
I'm not sure either if I'm honest, I've added a comment below, but I think it might be best to just try it on develop.